packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -45,7 +45,25 @@ const _allInstances: {
   [key: string]: FocusZone;
 } = {};
 
-const _outerZones: Set<FocusZone> = new Set();
+const outZones = {
+  _outerZones: new Map<Window, Set<FocusZone>>(),
+  register(window: Window, FZ: FocusZone) {
+    if (this._outerZones.get(window)) {
+      this._outerZones.get(window)?.add(FZ);
+    } else {
+      this._outerZones.set(window, new Set([FZ]));
+    }
+  },
+  unregister(window: Window, FZ: FocusZone) {
+    this._outerZones.get(window)?.delete(FZ);
+  },
+  deleteOutZone(window: Window) {
+    this._outerZones.delete(window);
+  },
+  getOutZone(window: Window) {
+    return this._outerZones.get(window);
+  },
```

Comment:
you can remove these 2 apis. And make sure unregister deletes the entry in the map when the last entry for the window is unregistered.

Reply:
removed `deleteOutZone` but `getOutZone` is only for tests usage